### PR TITLE
Lps 163054

### PR DESCRIPTION
**Pull request by slnn**
https://issues.liferay.com/browse/LPS-163054

/cc @tinatian @dantewang

### CODE REVIEW

**Comment by slnn** [comment]
Hi @catarineaguiar 

Could you help to scan test results when they are finished?

Thanks
Lily

File: modules/dxp/apps/saml/saml-opensaml-integration/src/main/java/com/liferay/saml/opensaml/integration/internal/servlet/profile/SingleLogoutProfileImpl.java [hunk 1]
```diff
@@ -426,6 +402,21 @@ public void terminateSsoSession(
 			SamlWebKeys.SAML_SSO_SESSION_ID, StringPool.BLANK, 0);
 	}
 
+	@Activate
+	protected void activate() {
+		super.setIdentifierGenerationStrategyFactory(
+			_identifierGenerationStrategyFactory);
+
+		super.metadataManager = _metadataManager;
```
Comment (dantewang) [inline modules/dxp/apps/saml/saml-opensaml-integration/src/main/java/com/liferay/saml/opensaml/integration/internal/servlet/profile/SingleLogoutProfileImpl.java:410]: @slnn For these, if their reference annotations don't have filters (target = xxx), you can put the references in the base class, and enable ds annotation inherit in bnd.bnd. Assigning them in activate method looks unusual.

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by slnn** [comment]
ci:test:relevant

**Comment by slnn** [comment]
ci:test:sf

File: modules/dxp/apps/saml/saml-opensaml-integration/src/main/java/com/liferay/saml/opensaml/integration/internal/servlet/profile/SingleLogoutProfileImpl.java (hunk 1 above)
Comment (slnn) [reply modules/dxp/apps/saml/saml-opensaml-integration/src/main/java/com/liferay/saml/opensaml/integration/internal/servlet/profile/SingleLogoutProfileImpl.java]: Hi @dantewang 

I have update and forced pushed to this pull/

Thanks
Lily

**Comment by catarineaguiar** [comment]
no regressions.

**Review by dantewang — APPROVED** [review]

**Comment by dantewang** [comment]
@shuyangzhou @tinatian Forwarding this one as all tests passed review and contains simple changes

**Comment by dantewang** [comment]
ci:forward

**Pull request closed without merging**